src/main/resources/swagger.api/schema/resource/title/TitleField.json

```diff
@@ -0,0 +1,37 @@
+{
+  "oneOf": [
+    {
+      "type": "object",
+      "title": "BasicTitleField",
+      "properties": {
+        "BasicTitle": {
+          "type": "object",
+          "$ref": "schema/resource/title/BasicTitle.json",
```

Comment:
I think "PrimaryTitle" is better name than "BasicTitle". 
"basic" do not indicate that this is the most important title of the Work.
"primary" clearly indicates that this is the important title of Work.

Reply:
agree, changed